client/icecc-create-env.in

```diff
@@ -6,10 +6,11 @@
 
 target_files=
 
-is_darwin=0
-if test `uname` = Darwin; then
-  is_darwin=1
-fi
+case `uname` in
+    "Darwin") is_darwin=1;;
+    "FreeBSD") is_freebsd=1;;
+    "Linux") is_linux=1;;
+esac
 
```

Comment:
We need to expand this for solarus, Open/net/dragonfly BSD, and probably a dozen others I can't think of...